sympy/sets/sets.py

```diff
@@ -1217,16 +1217,7 @@ def boundary_of_set(i):
         return Union(*map(boundary_of_set, range(len(self.args))))
 
     def _contains(self, other):
-        try:
-            d = Dummy()
-            r = self.as_relational(d).subs(d, other)
-            b = tfn[r]
-            if b is None and not any(isinstance(i.contains(other), Contains)
-                    for i in self.args):
-                return r
-            return b
-        except (TypeError, NotImplementedError):
-            return Or(*[s.contains(other) for s in self.args])
+        return Or(*[s.contains(other) for s in self.args])
```

Comment:
To me this doesn't feel right because `_contains` should return non-symbolic True, False but it's already an ambiguous method in that respect because it allows symbolic return values:
https://github.com/sympy/sympy/blob/4fc50841e38205561523e0a9d0e5da173988b04a/sympy/sets/sets.py#L344-L352